SSA/Projects/InstCombine/update_alive_statements.py

```diff
@@ -85,7 +85,7 @@ def getStatement(preamble: List[str], id : int, proof: List[str]) -> str:
 
     stmt = name
     stmt += msg
-    stmt += " := by\n  simp_alive_undef\n  simp_alive_ops\n  simp_alive_case_bash\n  try alive_auto\n  try sorry"
+    stmt += " := by\n  simp_alive_undef\n  simp_alive_ops\n  simp_alive_case_bash\n  try alive_auto\n  try sorry\n  try sorry"
```

Comment:
```suggestion
    stmt += " := by\n  simp_alive_undef\n  simp_alive_ops\n  simp_alive_case_bash\n  try alive_auto\n  all_goals sorry"
```
I think `all_goals` should already be fine with the case of 0 goals, otherwise we might want `all_goals (try sorry)` or some variation thereof